Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=3be36ae223271f9c2cfbe7406846c8fdcd2f50c3
Commit:     3be36ae223271f9c2cfbe7406846c8fdcd2f50c3
Parent:     51fb80fefe736db1182551fec6528d1ef095b0ea
Author:     Stefan Lippers-Hollmann <s.L-H@gmx.de>
AuthorDate: Sun Jan 4 01:10:49 2009 +0100
Committer:  John W. Linville <linville@tuxdriver.com>
CommitDate: Mon Jan 12 14:24:50 2009 -0500

    rt2x00: add USB ID for the Linksys WUSB200.
    
    add USB ID for the Linksys WUSB200 Wireless-G Business USB Adapter to
    rt73usb.
    
    Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
    Cc: stable <stable@kernel.org>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

 drivers/net/wireless/rt2x00/rt73usb.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/wireless/rt2x00/rt73usb.c b/drivers/net/wireless/rt2x00/rt73usb.c
index d638a8a..96a8d69 100644
--- a/drivers/net/wireless/rt2x00/rt73usb.c
+++ b/drivers/net/wireless/rt2x00/rt73usb.c
@@ -2321,6 +2321,7 @@ static struct usb_device_id rt73usb_device_table[] = {
 	/* Linksys */
 	{ USB_DEVICE(0x13b1, 0x0020), USB_DEVICE_DATA(&rt73usb_ops) },
 	{ USB_DEVICE(0x13b1, 0x0023), USB_DEVICE_DATA(&rt73usb_ops) },
+	{ USB_DEVICE(0x13b1, 0x0028), USB_DEVICE_DATA(&rt73usb_ops) },
 	/* MSI */
 	{ USB_DEVICE(0x0db0, 0x6877), USB_DEVICE_DATA(&rt73usb_ops) },
 	{ USB_DEVICE(0x0db0, 0x6874), USB_DEVICE_DATA(&rt73usb_ops) },
